examples/add_items_to_index.py

```diff
@@ -0,0 +1,42 @@
+"""
+This example shows how to add your items in autocomplete index
+Items used: /examples/items_for_indexation.csv
+
+More: https://constructor.io/docs/#autocomplete-items
+"""
+
+import os
+import csv
+import webbrowser
+from six.moves import input
+
+from constructor_io import ConstructorIO
+
+if __name__ == "__main__":
+    api_token = input("Enter api token: ")
+    autocomplete_key = input("Enter autocomplete key: ")
```

Comment:
We're calling this `key` now. The API still accepts `autocomplete_key` in the parameters for backwards compatibility, but it also accepts `key=...`, because we don't want to use the term "autocomplete" for things that are also used in search.